README.md

```diff
@@ -73,6 +73,14 @@ and start with either `fix/` or `feature/`. E.g. a branch name for adding
 the navigation menu might be `feature/324-nav-menu`.
 
 # Deployment 
+
+## Manual deployment
 For deployment to a CloudFoundry server simply [install the CLI](https://github.com/cloudfoundry/cli), log in and then `cf push`. The `manifest.yml` file
 can be tweaked if needed. Currently we are using a 'static file' buildpack. Note that
-only the `build` directory is deployed, so you should `npm build` before deploying.
\ No newline at end of file
+only the `build` directory is deployed, so you should `npm build` before deploying.
+
+## CI/CD
+CircleCI will test all commits pushed to the repo. As soon as a branch is merged
+to the `master` branch, it will (assuming tests pass) additionally deploy to the
+`smart-cities-dev` instance. To deploy to the `smart-cities-staging` instance,
+simply push to the `staging` branch.
```

Comment:
Do we know URLs at this point? Should we add them to the readme?

Reply:
I think probably best just to know this within the team as it's an open repo; not that we're explicitly _hiding_ our work in progress, but I don't see any value in putting those URLs out there.